src/classes/Environment.php

```diff
@@ -1821,6 +1821,61 @@ function( $stderr ) {
 		return true;
 	}
 
+	/**
+	 * WP CLI Runner
+	 *
+	 * @param string  $command A WP CLI command.
+	 * @return array
+	 */
+	public function wpCliRunner( $command ) {
+		Log::instance()->write( 'Running wp command: ' . $command, 0 );
+
+		$command = 'wp ' . $command . ' --allow-root';
+
+		$exec_config = new ContainersIdExecPostBody();
+		$exec_config->setTty( true );
+		$exec_config->setAttachStdout( true );
+		$exec_config->setAttachStderr( true );
+		$exec_config->setCmd( [ '/bin/bash', '-c', $command ] );
+
+		$exec_command      = EnvironmentFactory::$docker->containerExec( $this->environment_id . '-wordpress', $exec_config );
+		$exec_id           = $exec_command->getId();
+		$exec_start_config = new ExecIdStartPostBody();
+		$exec_start_config->setDetach( false );
+
+		$stream = EnvironmentFactory::$docker->execStart( $exec_id, $exec_start_config );
+		$output = '';
+		$error = '';
+
+		$stream->onStdout(
+			function( $_stdout ) use ( &$output ) {
+				$output .= $_stdout;
```

Comment:
I'd probably log the output here under high verbosity like we do for other exec calls.